content/community/element.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "Element"
```

Comment:
Best to mention "Matrix" the protocol instead of "Element" client. It is, though, great to suggest Element as a recommended client for Matrix, however there is an important (to our crowd) distinction there.

Reply:
That's an awesome clarrification, and I didn't even think about that :sweat_smile: 

Maybe we can even have a list of different clients that we could recommend in the MD content of the element page.

Reply:
Covered renaming:
https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/129/commits/69c9801e7f7dbd5c43e3285e1509e59761ac6eef